core/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@jahia/external-provider",
+  "version": "0.0.1",
+  "husky": {
+    "hooks": {
+      "pre-push": "yarn lint:fix"
+    }
+  },
+  "scripts": {
+    "test": "env-cmd --no-override jest",
+    "testcli": "jest",
+    "build": "yarn lint:fix && yarn webpack",
+    "build:nolint": "yarn webpack",
+    "dev": "yarn webpack --watch",
+    "webpack": "node --max_old_space_size=2048 ./node_modules/.bin/webpack",
+    "build:analyze": "yarn build --analyze",
+    "build:production": "yarn build -p",
+    "build:production-analyze": "yarn build -p --analyze",
+    "clean": "rimraf *.log src/main/resources/javascript/apps",
+    "clean:all": "yarn clean && rimraf node_modules node",
+    "lint": "./node_modules/.bin/eslint --ext js,jsx .",
+    "lint:fix": "./node_modules/.bin/eslint --ext js,jsx --fix ."
+  },
+  "main": "index.js",
+  "license": "MIT",
+  "dx-extends": {
+    "@jahia/jahia-ui-root": "0.0.1"
+  },
+  "dependencies": {
+    "@jahia/moonstone": "^0.6.0",
```

Comment:
Why do we need moonstone in those projects? As long as we do not use icons we don't need it (could not find any import from moonstone)

Reply:
I added it thinking that at some point the project will evolve into an app (it was actually a copy paste from existing project, I removed some stuff that was obviously not needed). The dependency doesn't get bundled so there is performance penalty or anything like that. But it can be removed and it won't affect anything.